### ISSUE TITLE
Map is really annoying to use due to the camera moving automatically.

### ISSUE THREAD

**Issue by Chambs-15** [SquareZeb/Interactive-Map-For-VOTV#3]
Whenever i try to zoom in and look at something in a corner, the window automatically re-centers, making it near impossible to check anything on the corners.

**Comment by SquareZeb** [comment]
Yeah the Leaflet API I am using is kind've weird I'll have to find a workaround for it.
